Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'D:\lami-accounts\node_modules\luxon\src\impl\formatter'

## Bug Report

`npx serverless offline` gives me the following error:

```
Error:
Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'D:\lami-accounts\node_modules\luxon\src\impl\formatter' imported from D:\lami-accounts\node_modules\luxon\src\info.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:404:11)
    at moduleResolve (node:internal/modules/esm/resolve:965:10)
    at defaultResolve (node:internal/modules/esm/resolve:1173:11)
    at nextResolve (node:internal/modules/esm/loader:173:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:852:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:439:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36)
```

I was following the following documentation on how to run a serverless Nest.js application offline : https://docs.nestjs.com/faq/serverless#example-integration

- file: serverless.yml

```yaml
service: my-service

plugins:
  - serverless-offline

provider:
  runtime: nodejs16.x
  stage: dev

functions:
  hello:
    events:
      - http:
          method: get
          path: hello
    handler: handler.hello
```

- file: handler.js
**Environment**

- `serverless` version: [e.g. v3.19.0]
- `serverless-offline` version: [e.g. v8.8.0]
- `node.js` version: [e.g. v18.3.0]
- `OS`: Windows 10

--- a/package.json
+++ b/package.json
@@ -200,7 +200,7 @@
     "jsonschema": "^1.4.1",
     "jsonwebtoken": "^8.5.1",
     "jszip": "^3.10.1",
-    "luxon": "^3.0.1",
+    "luxon": "3.0.1",
     "node-fetch": "^3.2.10",
     "node-schedule": "^2.1.0",
     "object.hasown": "^1.1.1",

